src/modules/esl-panel-group/core/esl-panel-group.shape.ts

```diff
@@ -21,6 +21,9 @@ export interface ESLPanelGroupTagShape extends ESLBaseElementShape<ESLPanelGroup
   /** Define a list of comma-separated "modes" to disable collapse/expand animation (for both Group and Panel animations)*/
   'no-animate'?: string;
 
+  /** Define active panels behaviour in case of mode changing. Supported values: `last|initial`*/
+  'refresh-state': string;
```

Comment:
Can it be not mandatory attribute with `initial` value by default?

Reply:
Exactly. Thanks